flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/util/collections/ByteHashSet.java

```diff
@@ -17,21 +17,17 @@
 
 package org.apache.flink.table.runtime.util.collections;
 
-import org.apache.flink.table.util.MurmurHashUtil;
-
 /**
  * Byte hash set.
  */
 public class ByteHashSet extends OptimizableHashSet {
 
-	private byte[] key;
-
-	private byte min = Byte.MAX_VALUE;
-	private byte max = Byte.MIN_VALUE;
+	private final byte min = Byte.MIN_VALUE;
```

Comment:
Remove useless field, directly use `Byte.MIN_VALUE`

Reply:
Sure. I have revised accordingly. Thanks for the suggestion.